README.md

```diff
@@ -35,7 +35,7 @@ The directories contain the following files:
 `go-tuf` is tested on Go versions 1.18.
 
 ```bash
-go get github.com/theupdateframework/go-tuf/cmd/tuf
+go install github.com/theupdateframework/go-tuf/cmd/tuf@latest
```

Comment:
I would recommend against suggesting latest and do something like

```
go install github.com/theupdateframework/go-tuf/cmd/tuf@${LATEST_VERSION}
```

to encourage people to find the latest release version (I suppose a link to releases works)

Reply:
I thought this is what latest does - https://go.dev/ref/mod#version-queries:~:text=The%20string%20latest%2C%20which%20selects%20the%20highest%20available%20release%20version

Reply:
Haha wow, my friday is going bad. yes! sorry i confused latest with main